release.py

```diff
@@ -493,7 +495,7 @@ def generate_upload_tarball(args):
     # If we're releasing under a different name, then rename the tarball (the
     # package itself doesn't know anything about this).
     if args.package != args.package_alias:
-        tarball_fname = '%s-%s.tar.bz2'%(args.package_alias, args.version)
+        tarball_fname = ' %s-%s.tar.bz2' % (args.package_alias, args.version)
```

Comment:
This failed for me with
```

 !! Can not find a tarball at: /tmp/tmp8ttiifnf/build/ gz-cmake-3.4.1.tar.bz2 or at /tmp/tmp8ttiifnf/build/ gz_cmake-3.4.1.tar.bz2

```

I think the issue is there are a number of places where an extra space is added inside string formats that didn't have it before. Search for `' %s`

Reply:
Good catch. Fixed in [38f04ab](https://github.com/gazebo-tooling/release-tools/pull/1014/commits/38f04ab0dcd479b13edd1906c311cc7418a86909) and [f82e937](https://github.com/gazebo-tooling/release-tools/pull/1014/commits/f82e937ec0b9920f81fcaeb691b764dc271d3b86)